tests/providers/openai/operators/test_openai.py

```diff
@@ -43,8 +65,53 @@ def test_execute_with_input_text():
 @pytest.mark.parametrize("invalid_input", ["", None, 123])
 def test_execute_with_invalid_input(invalid_input):
     operator = OpenAIEmbeddingOperator(
-        task_id="TaskId", conn_id="test_conn_id", model="test_model", input_text=invalid_input
+        task_id=TASK_ID, conn_id=CONN_ID, model="test_model", input_text=invalid_input
     )
     context = Context()
     with pytest.raises(ValueError):
         operator.execute(context)
+
+
+@pytest.mark.parametrize("wait_for_completion", [True, False])
```

Comment:
Hi @vincbeck, here is the change for testing `wait_for_completion`

Reply:
My bad!

Reply:
no worries 😄